.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+  "singleQuote": false,
+  "trailingComma": "none",
```

Comment:
Was this by purpose?
Why not having the es5 approach with trailing commas for objects and arrays?